lib/utils.js

```diff
@@ -142,3 +142,11 @@ exports.runManualTestCase = async (test, testCase) => {
   printInstructionsSet('CLEANUP', testCase.cleanup)
   test.true(await ipc.requestConfirmation())
 }
+
+exports.getDeviceUptime = (ssh) => {
+  const start = process.hrtime()[0]
+  return new Bluebird(async (resolve, reject) => {
```

Comment:
Why are we returning a whole promise? We can mark the `getDeviceUptime` function as `async` and then do:

```js
const uptime = await ssh('cut -d " " -f 1 /proc/uptime')
return Number(uptime) - (start - process.hrtime()[0])
```

Reply:
Good catch, this function used to be very very big and required a promise. Not the case anymore, clearly.